br/pkg/gluetidb/glue.go

```diff
@@ -231,6 +231,8 @@ func (gs *tidbSession) CreateTables(ctx context.Context, tables map[string][]*mo
 			cloneTables = append(cloneTables, table)
 		}
 		gs.se.SetValue(sessionctx.QueryString, queryBuilder.String())
+		// Disable foreign key check when batch create tables.
+		gs.se.GetSessionVars().ForeignKeyChecks = false
```

Comment:
it looks like we can set ForeignKeyChecks out of this loop.  my understanding is that ForeignKeyChecks is per session. only need to set it per session will be enough? correct me if I am wrong @crazycs520

Reply:
great, done.